packages/devtools_app/lib/src/screens/memory/framework/offline_data/offline_data.dart

```diff
@@ -32,29 +32,36 @@ class OfflineMemoryData with Serializable {
   });
 
   factory OfflineMemoryData.fromJson(Map<String, dynamic> json) {
-    return OfflineMemoryData(
-      deserialize<DiffPaneController>(
-        json[Json.diffData.name],
-        DiffPaneController.fromJson,
-      ),
-      deserializeNullable<ProfilePaneController>(
-        json[Json.profileData.name],
-        ProfilePaneController.fromJson,
-      ),
-      deserializeNullable<ChartData>(
-        json[Json.chartData.name],
-        ChartData.fromJson,
-      ),
-      deserializeNullable<TracePaneController>(
-        json[Json.trace.name],
-        TracePaneController.fromJson,
-      ),
-      deserialize<ClassFilter>(
-        json[Json.classFilter.name],
-        ClassFilter.fromJson,
-      ),
-      selectedTab: json[Json.selectedTab.name] as int? ?? 0,
+    final diff = deserialize<DiffPaneController>(
```

Comment:
why use deserialize here and deserializeNullable for others?

Reply:
diff is present of memory screen for each mode
other controllers are not.